cellxgene_schema_cli/cellxgene_schema/generate_sanity_schemas.py

```diff
@@ -30,12 +31,32 @@ def get_dataset_metadata_uns(adata, uns_keys):
         dm.append(v)
     return dict(zip(uns_keys, dm))
 
+def parse_publication_date(pub_date):
```

Comment:
let's use the dateutil package to be more robust to differently formated strings. Also I think we should raise the exception when we are not able to pass the date. If we are not able to parse it, we should fix it in the anndata object manually before uploading (indicate this in the error message). This way we can expect properly formatted dates for the website

Reply:
done!